.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: PR CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  pre_merge_ci:
+    name: "Lite CI"
```

We don't need the lite profile here (and seems we don't use it)
```suggestion
    name: "CI"
```